pkgs/tools/misc/hwatch/default.nix

```
@@ -0,0 +1,33 @@
+{ lib, stdenv, fetchFromGitHub, fetchpatch, rustPlatform }:
+
+rustPlatform.buildRustPackage rec {
+  pname = "hwatch";
+  version = "0.3.5";
+
+  src = fetchFromGitHub {
+    owner = "blacknon";
+    repo = pname;
+    rev = version;
+    sha256 = "sha256-dGntqLq+wDEsYU1HInX4g7De+WnDZcFpep/zxudlNSk=";
+  };
+
+  cargoPatches = [
+    # Adding missing Cargo.lock for build.
+    # Can be removed once fixed upstream: https://github.com/blacknon/hwatch/issues/53
+    (fetchpatch {
+      url = "https://github.com/blacknon/hwatch/pull/57/commits/c9584ca90bbc5d5ddef8e84defe35421e46c2251.patch";
+      sha256 = "sha256-j5g33pLgZWlxv4wuadCLc19wZPN/KNzPkW6Pfzx0O4o=";
+    })
+  ];
+  cargoSha256 = "sha256-cFpfdlItcbYtO6788OEKMm3nmEzfVFsdgrcF4mdlFwk=";
+
+  meta = with lib; {
+    homepage = "https://github.com/blackmon/hwatch";
+    description= "Modern alternative to the watch command";
+    longDescription = ''
+      A modern alternative to the watch command, records the differences in execution results and can check this differences at after
+    '';
```

Usually `longDescription` has a period at the end unlike `description`.

```suggestion
    longDescription = ''
      A modern alternative to the watch command, records the differences in
      execution results and can check this differences at after.
    '';
```